docs/guide/examples.rst

```diff
@@ -16,6 +16,22 @@ Train a Truncated Quantile Critics (TQC) agent on the Pendulum environment.
   model.learn(total_timesteps=10000, log_interval=4)
   model.save("tqc_pendulum")
 
+QR-DQN
```

Comment:
Please also update the changelog (put your username next to each entry) and add your username at the bottom ;)